generated/nidcpower/session.py

```diff
@@ -4151,6 +4151,599 @@ def export_attribute_configuration_file(self, file_path):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def create_advanced_sequence(self, sequence_name, sequence, set_as_active_sequence=True):
+        '''create_advanced_sequence
+
+        Test.
+
+        Note: Test.
+
+        Args:
+            sequence_name (str): Name of sequence.
+
+            sequence (list of int): Sequence.
+
+            set_as_active_sequence (bool): Specifies that this current sequence is active.
+
+        '''
+        attribute_lookup = {
```

Comment:
If it wasn't for the property lookups, I'd say this should be a constant outside of this functions (a `_ATTRIBUTE_LOOKUP`)

Reply:
Changed.